test/e2e/components/vl-button.js

```diff
@@ -66,18 +79,18 @@ class VlButton extends VlElement {
         return (await this.getPillType()) === 'error';
     }
 
+}
+
+class VlButtonInputAddon extends VlButtonElement {
```

Comment:
Is het een VlButtonElement of is het een VlInputAddon of is het beide?

Reply:
Het is een VlInputAddonElement maar met een `<button/>` tag, dus beter extenden van VlInputAddonElement.